src/waveform/widgets/waveformwidgetabstract.cpp

```diff
@@ -43,7 +42,7 @@ void WaveformWidgetAbstract::resize(int width, int height) {
     qreal devicePixelRatio = 1.0;
     if (m_widget) {
         m_widget->resize(width, height);
-        devicePixelRatio = getDevicePixelRatioF(m_widget);
+        devicePixelRatio = m_widget->devicePixelRatio();
```

Comment:
Please always check all occurrences, I only amend a single one!

Reply:
Whoops, this got wrapped up in some other accidental changes I made which I then reverted so this one wasn't taken care of.